sklearn/utils/tests/test_seq_dataset.py

```diff
@@ -6,17 +6,25 @@
 from numpy.testing import assert_array_equal
 import scipy.sparse as sp
 
-from sklearn.utils.seq_dataset import ArrayDataset, CSRDataset
+from sklearn.utils.seq_dataset import ArrayDataset64 as ArrayDataset
```

Comment:
It could be confusing and I would keep the 64 at the end and replace in the tests.